scripts/generate_records/README.md

```diff
@@ -1,22 +1,38 @@
-# Generate DB records
+# Generate Records
 
-This script (generate_db_records.js) is meant to push up large quantities of realistic cumulus database entries for scaled testing purposes
+These scripts (generate_db_records.js and generate_db_executions.js) are meant to push up large quantities of realistic cumulus database entries for scaled testing purposes
 
 ## Installation
 This can be installed with npm install in this directory (or will be installed as a part of cumulus when installing the whole of cumulus-core)
 
-generate_db_records.js is tested to run with both node v16.19.0 and v20.12.2
+generate_db_records.js and generate_db_executions.js are tested to run with both node v16.19.0 and v20.12.2
 
-## Configuration
+## generate_db_records.js
+This is the default script for uploading bulk data to a cumulus database. it is granule oriented, handling files, executions and granule-executions with respect to granules and as such is well optimized for most database mocking applications. It will not be performant at producing large numbers of executions, files, or granule-executions but not a large number of granules
```

Comment:
What does this mean?
>It will not be performant at producing large numbers of executions, files, or granule-executions but not a large number of granules

Reply:
https://github.com/nasa/cumulus/pull/3721/commits/76a8f4914a4b57b3ef6ea690ab0ecb7f6d823f5b
forgot about this question in all the other questions, but I've reworded it to be more clear